sympy/utilities/iterables.py

```diff
@@ -1381,6 +1381,12 @@ def partitions(n, m=None, k=None, size=False):
         raise ValueError("n must be >= 0")
     if m == 0:
         raise ValueError("m must be > 0")
+    if n == 0:
+        if size:
+            yield 1, {0: 1}
```

Comment:
I'm not sure about these. Normally partitions are expected to consist of positive integers. Maybe empty dictionary could be returned?